app/views/maintenance_tasks/tasks/show.html.erb

```diff
@@ -4,10 +4,26 @@
   <%= @task %> <%= status_tag(@task.status) %>
 </h1>
 
-<%= render 'maintenance_tasks/runs/info', run: @task.last_run, with_status: false if @task.last_run %>
+<%= render 'maintenance_tasks/runs/info', run: @last_run, with_status: false if @last_run %>
 
 <div class="buttons">
-  <%= render "maintenance_tasks/tasks/actions/#{@task.status}", task: @task %>
+  <% if @last_run.nil? || @last_run.completed? %>
+    <%= render 'maintenance_tasks/tasks/run_task', task: @task %>
+  <% elsif @last_run.cancelling? %>
+    <%= button_to 'Run', run_task_path(@task), method: :put, class: 'button is-success', disabled: true %>
+    <% if @last_run.stuck? %>
+      <%= button_to 'Cancel', cancel_task_run_path(@task, @last_run), method: :put, class: 'button is-danger', disabled: @task.deleted? %>
```

Comment:
@etiennebarrie I noticed that this `Cancel` action is still disabled depending on whether the task is deleted or not... Which makes sense because if it's stuck, the job is actually already terminated, it's just the status that hasn't been updated to reflect that, so there's no concern about actually needing to cancel it.

I'm wondering though if it makes sense to have the button pop up at all if the Task is deleted.. Maybe we check that first:
```ruby
 <% if !@task.deleted? && @last_run.stuck? %>
   <%= button_to 'Cancel', cancel_task_run_path(@task, @last_run), method: :put, class: 'button is-danger' %>
<% end %>
```

(This is a pretty specific situation and it's quite possible I'm overthinking it, I was just bothered by the lack of symmetry between the `Cancel` buttons and disliked that this was the only one that was conditionally disabled 😛 )

WDYT?

Reply:
I would definitely ignore for this PR. But I think we should definitely be able to do it, it was simply forgotten from #216.

Reply:
I'm okay to ignore, wasn't sure if it was forgotten or intentional to disable this. It's such an odd edge case too, I'm not sure it's even worth addressing, was just curious what you thought 😁